src/applications/gi-sandbox/constants.js

```diff
@@ -20,6 +20,38 @@ export const KEY_CODES = Object.freeze({
   enterKey: 13,
 });
 
+/**
+ * Mapbox init values
+ */
+export const MapboxInit = {
+  zoomInit: 3,
+  centerInit: {
+    lng: -99.27246093750001,
+    lat: 40.17887331434698,
+  },
```

Comment:
Why these magic init values?

Reply:
Centerish of USA, lat/long is for Turkey Creek, NE
Used in https://github.com/department-of-veterans-affairs/vets-website/pull/17247
![image](https://user-images.githubusercontent.com/1094999/118869901-f8566380-b8b3-11eb-81ac-4c5be4b2729c.png)

Reply:
Can you create some constants, so we know why they were picked?

Reply:
Added comment to explain lat/long and default zoom values